package.json

```diff
@@ -37,7 +37,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged && git diff --name-only --staged | grep '.rb$' | xargs bundle exec rubocop --except Metrics --auto-correct --format quiet app/controllers/application_controller.rb && git diff --name-only --staged | grep '.rb$' | xargs git add"
+      "pre-commit": "pretty-quick --staged && git diff --name-only --staged | grep '.rb$' | xargs bundle exec rubocop --except Metrics --auto-correct --format quiet --force-exclusion app/controllers/application_controller.rb && git diff --name-only --staged | grep '.rb$' | xargs git add"
```

Comment:
This forces rubocop to honour the Excludes list rather than honouring the files passed on the command line.